### ISSUE TITLE
Implement name-refactoring for the module import directives

### ISSUE THREAD

**Issue by Hungkhoaitay** [source-academy/js-slang#1079]
For example in this code:
``` {.}
import { unit_circle as u, draw_connected as d } from "curve";
d(1000)(u);
```

We want somehow that users can click on the 'u' and rename all occurrences to (for example) 'uc'.

**Comment by martin-henz** [comment]
Yes, this renaming (aka "refactoring") should be possible with Ctrl-M, see https://github.com/source-academy/frontend/wiki/Source-Academy-Keyboard-Shortcuts

**Comment by martin-henz** [comment]
Reminder: @hungkhoaitay you are probably uniquely qualified to fix this. If you have time.

**Comment by hungkhoaitay** [comment]
@martin-henz Yes prof, sorry that I couldn't finish it soon, I will try to find some time during recess week.
